examples/CSharp/CSharp.Examples/Scenarios/HelloWorld.cs

```diff
@@ -17,5 +17,28 @@ public static Scenario BuildScenario()
             });
             return ScenarioBuilder.CreateScenario("Hello World from NBomber!", step1);
         }
+
+        public static Scenario BuildPlainScenario()
+        {
+
+            //Just a sample how to create no-context step
+            var step1 = Step.Create("plain step", () => Task.FromResult(Response.Ok()));
+            return ScenarioBuilder.CreateScenario("Hello World from NBomber struggle continues!", step1);
+        }
+    }
+
+    class HelloWorld2Scenario

```

Comment:
let's modify all existent examples except WebSockets example
we don't need two HelloWorlds